include/modem/location.h

```diff
@@ -206,28 +209,28 @@ struct location_event_data {
 
 	/** Event specific data. */
 	union {
-		/** Current location, used with event LOCATION_EVT_LOCATION. */
+		/** Current location, used with event @ref LOCATION_EVT_LOCATION. */
 		struct location_data location;
 
 #if defined(CONFIG_LOCATION_DATA_DETAILS)
 		/**
 		 * Relevant location data when a timeout or an error occurs.
-		 * Used with event LOCATION_EVT_TIMEOUT and LOCATION_EVT_ERROR.
+		 * Used with event @ref LOCATION_EVT_TIMEOUT and @ref LOCATION_EVT_ERROR.
 		 */
 		struct location_data_error error;
 #endif
 
 #if defined(CONFIG_LOCATION_SERVICE_EXTERNAL) && defined(CONFIG_NRF_CLOUD_AGPS)
 		/**
-		 * A-GPS notification data frame used by GNSS to let the application know it
-		 * needs new assistance data, used with event LOCATION_EVT_GNSS_ASSISTANCE_REQUEST.
+		 * A-GPS notification data frame used by GNSS to let the application know it needs
```

Comment:
A-GPS -> A-GNSS?

Reply:
Will be done separately for the entire sdk-nrf